skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/NavansattService.kt

```diff
@@ -28,14 +26,42 @@ class NavansattService(config: Config, authService: AzureADService) {
         }
     }
 
-    suspend fun hentNavAnsattEnhetListe(call: ApplicationCall, ansattId: String): ServiceResult<List<NAVEnhet>, String> {
-        return client.get(call,"navansatt/$ansattId/enheter").toServiceResult<List<NAVEnhet>, String>()
+    suspend fun harAnsattTilgangTilEnhet(
+        call: ApplicationCall,
+        ansattId: String,
+        enhetNr: String
+    ): Boolean {
+        val result = hentNavAnsattEnhetListe(call, ansattId)
+        return if (result is ServiceResult.Ok) {
```

Comment:
Forslag: return result is ServiceResult.Ok && result.result.enheter.any { it.enhetNr == enhetNr }